BUILD.gn

```diff
@@ -116,7 +116,8 @@ group("flutter") {
     ]
   }
 
-  if (is_ios || is_android) {
+  if ((flutter_runtime_mode == "debug" || flutter_runtime_mode == "profile") &&
+      (is_ios || is_android)) {
```

Comment:
Since linting is now part of this target, make sure we're not unnecessarily building it.

@zanderso

Reply:
is `profile` used somewhere?  As far as I remember, debug is only used.

Reply:
It's used for firebase testlab

Reply:
How much time does linting add to these build configurations?